### PR TITLE
Fixed broken link to keyless signing

### LINKED ISSUE #202: Broken Link to Keyless Signatures
**Description**

On page "[cosign --> Signing Containers](https://docs.sigstore.dev/cosign/signing_with_containers/)" ([file](https://github.com/sigstore/docs/blob/main/content/en/cosign/signing_with_containers.md)) the link to "Keyless Signatures" ([link](https://docs.sigstore.dev/cosign/keyless)) is broken and returns a HTTP404.

I _assume_ it shall point to "[cosign --> OpenID Connect signing](https://docs.sigstore.dev/cosign/openid_signing/)" ([file](https://github.com/sigstore/docs/blob/main/content/en/cosign/openid_signing.md)) instead.

**Version**

Cosign Public Documentation

### PR DESCRIPTION
**Pull request by smythp**
Resolves #202

### CODE REVIEW

File: content/en/cosign/signing_with_containers.md [hunk 1]
```diff
@@ -230,9 +230,9 @@ To use a AWS KMS CMK (Custom Master Key) for signing and verification, first cre
 
 ```shell
 $ export AWS_CMK_ID=$(aws kms create-key --customer-master-key-spec RSA_4096 \
-                                   --key-usage SIGN_VERIFY \
-                                   --description "Cosign Signature Key Pair" \
-                                   --query KeyMetadata.KeyId --output text)
+								   --key-usage SIGN_VERIFY \
+								   --description "Cosign Signature Key Pair" \
+								   --query KeyMetadata.KeyId --output text)
```
Comment (cpanato) [inline content/en/cosign/signing_with_containers.md:235]: i think you need to remove the tabs and not add it

File: content/en/cosign/signing_with_containers.md [hunk 2]
```diff
@@ -183,13 +183,13 @@ Sign the payload with `gcloud kms`:
...
-      --keyring=foo \
-      --version=1 \
-      --location=us-central
+	  --digest-algorithm=sha256 \
+	  --input-file=payload.json \
+	  --signature-file=gcpkms.sig \
+	  --key=foo \
+	  --keyring=foo \
+	  --version=1 \
+	  --location=us-central
```
Comment (cpanato) [inline content/en/cosign/signing_with_containers.md:192]: i think you need to remove the tabs and not add it

File: content/en/cosign/signing_with_containers.md [hunk 3]
```diff
@@ -245,11 +245,11 @@ Sign the payload with the AWS KMS CMK we created above:
...
-              --message file://payload.json \
-              --message-type RAW \
-              --signing-algorithm RSASSA_PKCS1_V1_5_SHA_256 \
-              --output text \
-              --query Signature > payload.sig
+			  --message file://payload.json \
+			  --message-type RAW \
+			  --signing-algorithm RSASSA_PKCS1_V1_5_SHA_256 \
+			  --output text \
+			  --query Signature > payload.sig
```
Comment (cpanato) [inline content/en/cosign/signing_with_containers.md:252]: same here

File: content/en/cosign/signing_with_containers.md [hunk 4]
```diff
@@ -275,12 +275,12 @@ Verify with AWS KMS using the CMK key we created in the first step:
...
-               --signing-algorithm RSASSA_PKCS1_V1_5_SHA_256 \
-               --signature fileb://remote_payload.sig \
-               --output text \
-               --query SignatureValid
+			   --message file://remote_payload.json \
+			   --message-type RAW \
+			   --signing-algorithm RSASSA_PKCS1_V1_5_SHA_256 \
+			   --signature fileb://remote_payload.sig \
+			   --output text \
+			   --query SignatureValid
```
Comment (cpanato) [inline content/en/cosign/signing_with_containers.md:283]: same here

File: content/en/cosign/signing_with_containers.md [hunk 5]
```diff
@@ -316,27 +316,27 @@ $ crane manifest $(cosign triangulate $IMAGE) | jq .
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
   "config": {
-    "mediaType": "application/vnd.docker.container.image.v1+json",
-    "size": 342,
-    "digest": "sha256:f5de0db6e714055d48b4bb3a374e9630c4923fa704d9311da6a2740cf625aaba"
+	"mediaType": "application/vnd.docker.container.image.v1+json",
+	"size": 342,
+	"digest": "sha256:f5de0db6e714055d48b4bb3a374e9630c4923fa704d9311da6a2740cf625aaba"
```
Comment (cpanato) [inline content/en/cosign/signing_with_containers.md:321]: same here

File: content/en/cosign/signing_with_containers.md [hunk 6]
```diff
@@ -316,27 +316,27 @@ $ crane manifest $(cosign triangulate $IMAGE) | jq .
...
+	  }
+	},
+	{
+	  "mediaType": "application/vnd.dev.cosign.simplesigning.v1+json",
+	  "size": 219,
+	  "digest": "sha256:583246418c2afd5bfe29694793d07da37ffd552aadf8879b1d98047178b80398",
+	  "annotations": {
+		"dev.cosignproject.cosign/signature": "MEUCIF/+szLKKA2q2+c86AXeWR7UeD5yYpW7p0waHordxNjhAiEAm5e+Hm7Jhv9JpSwHpTc6aGLSkL6/Acm/z+b8mhfGXqY="
+	  }
+	}
```
Comment (cpanato) [inline content/en/cosign/signing_with_containers.md:339]: same here

**Review by cpanato — CHANGES_REQUESTED** [review]
thanks for the PR
some things that i think is a bit off

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thanks @smythp ! Could you make the changes that @cpanato requested here? Thanks!

**Review by ltagliaferri — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Pull request merged**